utils/walkerArgs.py

```diff
@@ -152,7 +152,7 @@ def parseArgs():
     parser.add_argument('-wh', '--webhook', action='store_true', default=False,
                         help='Activate webhook support')
     parser.add_argument('-whurl', '--webhook_url', default='',
```

Comment:
I think you can make multiple webhook urls easier to setup if you add ` action='append'` to the parser argument definition.

Then you can simply do: 

```
webhook_url: [mon|weather|raid]http://example.org/foo/bar 
webhook_url: [raid]http://otherUrl.org/foo/bar 
```
for a less cluttered way of configuration.

Reply:
I like the idea, thing is: what's easier to explain? The extra line or the comma seperated list?

Reply:
I'd say it's easier to just add lines, the users who need more than one would probably be techy enough to figure it out any which way

Reply:
Okay, aggreed. @temp666 if you like, feel free to PR this as well, otherweise I'm going to work on it next week.